kafka/conn.py

```diff
@@ -251,67 +251,42 @@ def __init__(self, host, port, afi, **configs):
         self._sasl_auth_future = None
         self.last_attempt = 0
         self._gai = None
-        self._gai_index = 0
         self._sensors = None
         if self.config['metrics']:
             self._sensors = BrokerConnectionMetrics(self.config['metrics'],
                                                     self.config['metric_group_prefix'],
                                                     self.node_id)
 
+    def _next_afi_host_port(self):
+        if not self._gai:
+            self._gai = dns_lookup(self._init_host, self._init_port, self._init_afi)
+            if not self._gai:
+                log.error('DNS lookup failed for {0}:{1} ({2})'
+                          .format(self._init_host, self._init_port, self._init_afi))
```

Comment:
Should probably be:

log.error('DNS lookup failed for %s:%i (%s)', self._init_host, self._init_port, self._init_afi)